bin/irida_fasta.py

```diff
@@ -27,24 +27,41 @@ def init_parser():
 def generate_samplelist(sample_tsv, directory):
 
     df_out = pd.DataFrame(columns=['Sample_Name', 'Project_ID', 'File_Forward', 'File_Reverse'])
+    input_format = ['sample', 'run', 'barcode', 'project_id', 'ct']
 
     with open(sample_tsv) as input_handle:
 
         for index, line in enumerate(input_handle):
 
             current_line_list = line.strip('\n').split('\t') # Order is [sample, run, barcode, project_id, ct]
 
-            # Error Checking Formatting
+
+            # Error checking for formatting
             if index == 0:
-                if str(current_line_list[0]) != 'sample':
-                    print('ERROR: First column of the header line must be called "sample". Exiting')
-                    quit() 
+                if len(current_line_list) != 5:
+                    if len(current_line_list) <= 4:
+                        print('ERROR: Header formated incorrectly. Please address this by matching the format: {}'.format(input_format))
+                        quit()
+
```

Comment:
add comment why we are doing trim all other rows.

Reply:
Added, good idea